ruby.c

```diff
@@ -2302,7 +2302,8 @@ process_options(int argc, char **argv, ruby_cmdline_options_t *opt)
 #endif
 #if USE_YJIT
     if (FEATURE_SET_P(opt->features, yjit)) {
-        opt->yjit = true; // set opt->yjit for Init_ruby_description() and calling rb_yjit_init()
+        bool rb_yjit_option_disable(void);
+        opt->yjit = !rb_yjit_option_disable(); // set opt->yjit for Init_ruby_description() and calling rb_yjit_init()
```

Comment:
Now that you have this, could you change https://github.com/ruby/ruby/blob/f7b53a75b648e7156f49c1d5c266e2d85f159fc6/yjit/src/yjit.rs#L37 to just `if yjit_enabled {`? The fact that we still have the duplicated check in `rb_yjit_init()` confuses me.

Reply:
Seconded

Reply:
Done.